src/python/pants/backend/python/subsystems/repos.py

```diff
@@ -60,6 +63,39 @@ class PythonRepos(Subsystem):
         advanced=True,
     )
 
+    path_mappings = StrListOption(
+        help=softwrap(
+            f"""
+            Mappings to facilitate using local Python requirements when the absolute file paths
+            are different on different users' machines. For example, the
+            path `file:///Users/pantsbuild/prebuilt_wheels/django-3.1.1-py3-none-any.whl` could
+            become `file://${{WHEELS_DIR}}/django-3.1.1-py3-none-any.whl`, where each user can
+            configure what WHEELS_DIR points to on their machine.
+
+            Expects values in the form `NAME|PATH`. You can specify multiple entries in the list.
```

Comment:
Is leaving off mention of the optional trailing `|DESCRIPTION` intentional?

Reply:
I figured to keep it simpler by not mentioning. The benefit of `|DESCRIPTION` is better error messages when `--path-mappings` is not defined at consumption side?

Reply:
I'm fine with keeping it simpler. If you have 4 find links repos, presumably it would be helpful if reading lock file in addition to error messages. Maybe not a worthwhile feature to have added though.